src/function/FunctionDocument.js

```diff
@@ -0,0 +1,58 @@
+import { XMLDocument } from 'substance'
+import FunctionSchema from './FunctionSchema'
+
+export default class FunctionDocument extends XMLDocument {
+
+  getDocTypeParams() {
+    return FunctionSchema.getDocTypeParams()
+  }
+
+  getXMLSchema() {
+    return FunctionSchema
+  }
+
+  getRoot() {
+    return this.get('function-1')
```

Comment:
I want to allow to control ids for certain elements. So this will be better in future.